queue_job/migrations/10.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+def migrate(cr, version):
+    if not version:
+        return
+    # In order to migrate connector from v. 9.0 to v. 10.0, we need to set
+    # connector module state to 'to upgrade'
```

Comment:
I suggest adding, 'If this is not done, all connector.* xmlids are removed due to the module renaming done by OpenUpgrade. In the future the approach sketched in https://github.com/OCA/queue/pull/23#issuecomment-325706811 may provide a more generic solution.'